tests/integration/polygonParts/polygonParts.spec.ts

```diff
@@ -5545,491 +5670,559 @@ describe('polygonParts', () => {
-          const response = await requestSender.findPolygonParts({
-            params: { polygonPartsEntityName: 'a' as EntityIdentifier },
-            body: { filter: featureCollection<Polygon | MultiPolygon>([]) },
-          });
+        expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
+        expect(response).toSatisfyApiSpec();
 
-          expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
-          expect(response).toSatisfyApiSpec();
+        expect.assertions(2);
+      });
 
-          expect.assertions(2);
+      it('should return 400 status code if polygonPartsEntityName is an invalid value - must follow a regex pattern (not more than 63 chars)', async () => {
+        const response = await requestSender.findPolygonParts({
+          params: { polygonPartsEntityName: 'a'.repeat(64) as EntityIdentifier },
+          body: { filter: featureCollection<Polygon | MultiPolygon>([]) },
         });
 
-        it('should return 400 status code if polygonPartsEntityName is an invalid value - must follow a regex pattern (not more than 63 chars)', async () => {
-          const response = await requestSender.findPolygonParts({
-            params: { polygonPartsEntityName: 'a'.repeat(64) as EntityIdentifier },
-            body: { filter: featureCollection<Polygon | MultiPolygon>([]) },
-          });
+        expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
+        expect(response).toSatisfyApiSpec();
 
-          expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
-          expect(response).toSatisfyApiSpec();
+        expect.assertions(2);
+      });
 
-          expect.assertions(2);
+      it('should return 400 status code if polygonPartsEntityName is an invalid value - must end with raster product type', async () => {
+        const response = await requestSender.findPolygonParts({
+          params: { polygonPartsEntityName: 'invalid_name' as EntityIdentifier },
+          body: { filter: featureCollection<Polygon | MultiPolygon>([]) },
         });
 
-        it('should return 400 status code if polygonPartsEntityName is an invalid value - must end with raster product type', async () => {
-          const response = await requestSender.findPolygonParts({
-            params: { polygonPartsEntityName: 'invalid_name' as EntityIdentifier },
-            body: { filter: featureCollection<Polygon | MultiPolygon>([]) },
-          });
+        expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
+        expect(response).toSatisfyApiSpec();
 
-          expect(response.status).toBe(httpStatusCodes.BAD_REQUEST);
-          expect(response).toSatisfyApiSpec();
+        expect.assertions(2);
+      });
 
-          expect.assertions(2);
+      it('should return 400 status code if polygonPartsEntityName is an invalid value - must resolve to a valid resource identifier (no longer than 63 chars)', async () => {
+        const customConfig: IConfig = {
```

Comment:
lets implement a real config mock outside this specific test, so we can reuse later when we  need, if we keep it like that, next test that will need custom config will duplicated this implementation and that something we dont want

Reply:
mocked the config